src/Microsoft.AspNetCore.SystemWebAdapters.CoreServices/RemoteAppClientExtensions.cs

```diff
@@ -73,6 +77,37 @@ public static ISystemWebAdapterRemoteClientAppBuilder Configure(this ISystemWebA
         return builder;
     }
 
+    /// <summary>
+    /// <see cref="HttpClient.BaseAddress"/> is set automatically, but if this is supposed to have a path, then that gets
+    /// lost. This handler will append that back if necessary.
```

Comment:
There's a way to make the request that avoids this issue. It will take some explaining, let's talk offline to see if it's any easier than this.